Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/DocumentPropertyEditor/Resources/OverrideResources.qrc

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE RCC><RCC version="1.0">
+    <qresource prefix="/OverrideResources">
+		<file>component_modified_as_override.svg</file>
+		<file>component_added_as_override.svg</file>
+    </qresource>
+</RCC>
```

Comment:
I was considering using the color inside the property row (unselected state) to be the color of the border. Do you have an example of what it looks like with the darker color? Do you like it? FYI the icon from above is not centered, so please don't use the exact same image as it will look off centered. You can get the centered on in the same location as the original icons provided from above.

![2023-02-28_14-08-31](https://user-images.githubusercontent.com/82551958/221992720-ae0450ac-a195-49ea-a8d8-b393cf356269.png)

Reply:
I had this thought because I noticed in your prototype it has the black border. But yeah I need to prototype it first so we can see if it can fit well.